lib/depject/page/html/render/profile.js

```diff
@@ -142,7 +154,10 @@ exports.create = function (api) {
     ])
 
     const prepend = h('header', { className: 'ProfileHeader' }, [
-      h('div.image', api.about.html.image(id)),
+      h('div.image', [
+        api.about.html.image(id),
+        h('div.lastActivity', computed(lastActivity, msg => `Last activity: ${api.message.html.timestamp(msg, false)}`))
```

Comment:
Add translate-ability to "Last activity" like this https://github.com/ssbc/patchwork/blob/33bda27b8cca77badcd0c336d1d6bf580c8ff077/lib/depject/page/html/render/settings.js#L48